packages/server/modules/multiregion/utils/dbSelector.ts

```diff
@@ -223,7 +223,16 @@ const setUpUserReplication = async ({
           info: { pubName, regionName }
         }
       )
-    if (!err.message.includes('already exists')) throw err
+    }
+
+    const errorMessage = err.message
+
+    if (
+      !['already exists', 'violates unique constraint'].some((message) =>
+        errorMessage.includes(message)
+      )
```

Comment:
just double check there is not a better way to make this check.
Like using an error code

Reply:
this does appear our best option in this case for now